code/modules/assembly/flash.dm

```diff
@@ -114,7 +114,7 @@
 		var/mob/living/silicon/robot/R = M
 		add_logs(user, R, "flashed", src)
 		update_icon(1)
-		M.Weaken(6)
+		M.Weaken(2)

```

Comment:
I believe 3 was the old value (Possibly set by Yackemflam). While I do not feel that flashes should stun for as long as they do, they should stun long enough for someone to be able to quickly emag a borg or yank its cell.